legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/toDDL.pure

```diff
@@ -101,13 +101,13 @@ function meta::relational::functions::database::loadCsvDataToDbTable(csvData:Lis
                                 | let schemaName = $tab.p.values->at(0).values->at(0);
                                   let tableName = $tab.p.values->at(1).values->at(0);
                                   let columnNames = $tab.p.values->at(2).values;
-                                  let schema = $db->schema($schemaName);
-                                  assertNotEmpty($schema, | 'Schema "' + $schemaName + '" not found in database "' + $db->elementToPath() + '"');
+                                  let schema = $db->map(d|$d->schema($schemaName))->toOne();
+                                  assertNotEmpty($schema, | 'Schema "' + $schemaName + '" not found in database "' + $db->at(0)->elementToPath() + '"');
```

Comment:
i think we should list out all the db names if the schema is not found not just the first one.